Modules/Filtering/ImageGrid/test/itkResampleImageTest7.cxx

```diff
@@ -0,0 +1,186 @@
+    iter.Set(value);
+    }
+
+  // Create an affine transformation
+  AffineTransformType::Pointer aff = AffineTransformType::New();
+  aff->Scale(0.9);
+
+  // Create a linear interpolation image function
+  InterpolatorType::Pointer interp = InterpolatorType::New();
+  interp->SetInputImage(image);
+
+  // Create and configure a resampling filter
+  itk::ResampleImageFilter< ImageType, ImageType >::Pointer resample =
+    itk::ResampleImageFilter< ImageType, ImageType >::New();
+
+  EXERCISE_BASIC_OBJECT_METHODS( resample, ResampleImageFilter, ImageToImageFilter );
+
+  resample->SetInterpolator(interp);
+
+  resample->SetInput(image);
+  TEST_SET_GET_VALUE( image, resample->GetInput() );
+
+  resample->SetSize(size);
+  TEST_SET_GET_VALUE( size, resample->GetSize() );
+
+  resample->SetTransform(aff);
+  TEST_SET_GET_VALUE( aff, resample->GetTransform() );
+
+  resample->SetInterpolator(interp);
+  TEST_SET_GET_VALUE( interp, resample->GetInterpolator() );
+
+  index.Fill( 0 );
+  resample->SetOutputStartIndex( index );
+  TEST_SET_GET_VALUE( index, resample->GetOutputStartIndex() );
+
+  ImageType::PointType origin;
+  origin.Fill( 0.0 );
+  resample->SetOutputOrigin( origin );
+  TEST_SET_GET_VALUE( origin, resample->GetOutputOrigin() );
+
+  ImageType::SpacingType spacing;
+  spacing.Fill( 1.0 );
+  resample->SetOutputSpacing( spacing );
+  TEST_SET_GET_VALUE( spacing, resample->GetOutputSpacing() );
+
+  using MonitorFilter = itk::PipelineMonitorImageFilter<ImageType>;
+  MonitorFilter::Pointer monitor = MonitorFilter::New();
+
+  using StreamerType = itk::StreamingImageFilter<ImageType,ImageType>;
+  StreamerType::Pointer streamer = StreamerType::New();
+
+  std::cout << "Test with normal AffineTransform." << std::endl;
+  monitor->SetInput( resample->GetOutput() );
+  streamer->SetInput( monitor->GetOutput() );
+
+  unsigned char numStreamDiv;
+
+  // Run the resampling filter without streaming, i.e. 1 StreamDivisions
+  numStreamDiv= 1; // do not split, i.e. do not stream
+  streamer->SetNumberOfStreamDivisions(numStreamDiv);
+  TRY_EXPECT_NO_EXCEPTION( streamer->Update() );
+
+  if (!monitor->VerifyAllInputCanStream(numStreamDiv))
+    {
+    std::cout << "Avoiding streaming failed to execute as expected!" << std::endl;
+    std::cout << monitor;
+    return EXIT_FAILURE;
+    }
+
+  ImagePointerType outputNoSDI= streamer->GetOutput(); // save output for later comparison
+  outputNoSDI->DisconnectPipeline(); // disconnect to create new output
+
+  // Run the resampling filter with streaming
+  numStreamDiv= 8; // split into numStream pieces for streaming.
+  resample->Modified(); // enforce re-execution even though nothing of filter changed
+  streamer->SetNumberOfStreamDivisions(numStreamDiv);
+  TRY_EXPECT_NO_EXCEPTION( streamer->Update() );
+
+  if (!monitor->VerifyAllInputCanStream(numStreamDiv))
+    {
+    std::cout << "Streaming failed to execute as expected!" << std::endl;
+    std::cout << monitor;
+    return EXIT_FAILURE;
```

Comment:
I'd suggest adding a
```
std::cerr << "Test failed!" << std::endl;
```
message and redirecting the output to `std::cerr` for the sake of consistency.

Reply:
Done.